docs/v1/openapi.yaml

```diff
@@ -666,6 +666,32 @@ paths:
           in: query
           required: true
           description: state
+  /oidc/present:
+    post:
+      summary: Used by verifier applications to initiate OpenID presentation flow through VCS
```

Comment:
to submit authorization response to verifier through VCS

Reply:
done